api/MediaKeySystemConfiguration.json

```diff
@@ -30,10 +30,10 @@
             "version_added": null
           },
           "safari": {
-            "version_added": null
+            "version_added": "13.1"
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/MediaKeySystemAccess/getConfiguration#browser_compatibility says Safari 12, which if true would mean that it returned something else than `MediaKeySystemConfiguration` in Safari 12 and 12.1. I suspect one is wrong, can you check?

Reply:
I was looking at the wrong file, admittedly -- I've corrected it now!